src/content/blog/whats-new-february-2024.mdx

```diff
@@ -0,0 +1,195 @@
+---
+title: "What's new in Astro - February 2024"
+description: "We started of 2024 strong"
+publishDate: "February 29, 2024"
+authors:
+  - elian
+coverImage: "/src/content/blog/_images/whats-new-shared.webp"
+socialImage: "/src/content/blog/_images/whats-new-february-2024/og.webp"
+lang: "en"
+---
+import ShowcaseGrid from './_whats-new-components/ShowcaseGrid.astro';
+import ThemeGrid from './_whats-new-components/ThemeGrid.astro';
+
+little sentence here @todo:
+
+## Updates from Mission Control
+
+- We have a [new channel on Discord](/chat) `#events`, where you can share events happening, share when you're live, and more.
+
+## Releases
+
+We're getting things ready for the launch of [Astro Studio](https://studio.astro.build).
+
+- [Astro 4.3](https://astro.build/blog/astro-430/)
+- [Astro 4.4](https://astro.build/blog/astro-440/) 
+- [Starlight 0.18](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.18.0) 
+- [Starlight 0.19](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.19.0)
+- [Starlight 0.20](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.20.0)
+
+## Docs
+
+@sarah to fill out!
+
+## Community
+
+### Welcome to our new Maintainers
+
+Both Fryuni and Oliver have joined Astro as Maintainers!
```

Comment:
Add liruifengv, and let's add GitHub links maybe?

Reply:
Ah great! Maybe better idea to link astro badges?